testdata/messages/reviewcomment/secrets/secret_review_content_no_ca_simplified.md

```diff
@@ -0,0 +1,43 @@
+
+
+---
+## 🗝️ Secret Detected
+
+---
+| Severity                | Finding                  |
+| :---------------------: | :-----------------------------------: |
+| Medium | Secret keys were found |
```

Comment:
The new default for secrets in now `High` and not `Medium` - please change the tests accordingly

Reply:
Its just a snapshot and example...